tests/pkg_lvgl/Makefile

```diff
@@ -0,0 +1,13 @@
+BOARD ?= stm32f429i-disc1
+include ../Makefile.tests_common
+
+# No interactive_sync
+DISABLE_MODULE += test_utils_interactive_sync
+
+USEPKG += lvgl
+USEMODULE += xtimer
+
+USEMODULE += ili9341
+CFLAGS += -DILI9341_LE_MODE
```

Comment:
You can remove this define if you let LittleVGL switch the endianness of the buffer (with `LV_COLOR_16_SWAP`). That significantly speeds up the SPI transfer throughput since the ili9341 doesn't have to swap every `uint16_t` before transfering allowing the SPI peripheral to fully utilize DMA for the whole buffer transfer.